<?xml version="1.0" encoding="utf-8"?>
<testsuite failures="1" name="docker_plugin_remove" skipped="0" tests="1" time="0.89" timestamp="2022-04-19T14:20:14">
	<testcase classname="DockerPluginRemoveTest" name="docker_plugin_remove">
		<failure message="FAILED: Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove/Output/linux/docker_images.out does not contain &quot;registry&quot; failed with 2 matches, first is: &quot;registry            2.7.1               e8a56425636e        5 months ago        22.2MB&quot;" type="FAILED"/>
		<system-out>14:20:13 INFO  ==============================================================
14:20:13 INFO  Id:    docker_plugin_remove
14:20:13 INFO  Dir:   plugin_docker/
14:20:13 INFO  Title: Validate docker plugin remove
14:20:13 INFO  ==============================================================
14:20:14 INFO  Executed sudo&lt;docker_images&gt;, exit status 0
14:20:14 WARN  Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove/Output/linux/docker_images.out does not contain &quot;registry&quot; failed with 2 matches, first is: &quot;registry            2.7.1               e8a56425636e        5 months ago        22.2MB&quot; ... failed 
14:20:14 WARN  Aborted test due to FAILED outcome
14:20:14 INFO  
14:20:14 INFO  cleanup:
14:20:14 INFO  
14:20:14 INFO  Test duration: 0.89 secs
14:20:14 INFO  Test final outcome:  FAILED
14:20:14 INFO  Test outcome reason: Grep on /home/pi/actions-runner/_work/thin-edge.io/thin-edge.io/tests/PySys/plugin_docker/docker_plugin_remove/Output/linux/docker_images.out does not contain &quot;registry&quot; failed with 2 matches, first is: &quot;registry            2.7.1               e8a56425636e        5 months ago        22.2MB&quot;
14:20:14 INFO  
</system-out>
	</testcase>
</testsuite>
